circuit/src/main/kotlin/com/slack/circuit/Presenter.kt

```diff
@@ -18,19 +18,22 @@ package com.slack.circuit
 import android.os.Parcelable
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.mutableStateOf
+import kotlinx.coroutines.flow.SharedFlow
 
 /**
  * A simple function that can [render] [UiState]. Usually sugar over [Ui.render] but defined
  * separately to keep the bones of the framework flexible.
  */
-fun interface StateRenderer<UiState, UiEvent : Any> where UiState : Any, UiState : Parcelable {
-  /** Renders a given [state] and emits [UI events][UiEvent] back up to [uiEvents]. */
-  @Composable fun render(state: UiState, uiEvents: (UiEvent) -> Unit)
+interface StateRenderer<UiState, UiEvent : Any> where UiState : Any, UiState : Parcelable {
```

Comment:
We should be able to remove this indirection entirely now

Reply:
Yup, removed